### ISSUE TITLE
Missing files in kafka microservice streaming example

### ISSUE THREAD

**Issue by ksbalan2018** [confluentinc/kafka-streams-examples#144]
I have been trying to run the micro service example locally and I noticed that files like Orderstate , product are missing from the repo. Is it a mistake in packaging the code or I am missing something

**Comment by Miro-Marinov** [comment]
@ksbalan2018 I think this is because on the `master` branch the parent pom's version atm is set to `5.1.0-SNAPSHOT` which cannot be resolved because it's not available in the confluent's maven repo. I'd suggest trying the following:

1. Make sure you are on branch `5.0.0-post` (or whatever the current latest `-post` version is)
2. Run `mvn clean install -Dmaven.test.skip=true`

**Comment by yonigibbs** [comment]
I executed `mvn clean install -Dmaven.test.skip=true` but still no luck I'm afraid.  Looks like the `io.confluent.examples.streams.avro` package is missing from the project, even when on the `5.0.0-post` branch.  I'm using IntelliJ, in case that makes a difference.

Where exactly should this package be?  Is part of the build process meant to create it?  If so, is it meant to generate a JAR, or some actual source files?  Just so I know where to look to see if I can find the generated artefacts in case I need to manually add them to my project or something.

Thanks.

**Comment by guozhangwang** [comment]
@yonigibbs ' issue has been resolved in https://github.com/confluentinc/kafka-streams-examples/issues/160 and I believe @Miro-Marinov 's response resolves @ksbalan2018 's issue, I'm closing this ticket for now.

**Comment by guozhangwang** [comment]
@yonigibbs just to confirm I did the same command as above and did not encounter the reported issue.

```
@ java -version
java version "1.8.0_66"
Java(TM) SE Runtime Environment (build 1.8.0_66-b17)
Java HotSpot(TM) 64-Bit Server VM (build 25.66-b17, mixed mode)

@ scala -version
Scala code runner version 2.11.7 -- Copyright 2002-2013, LAMP/EPFL
```

**Comment by yonigibbs** [comment]
Cool, thanks very much for looking into this.  I did a bit more investigation, and now think I've got to the bottom of this issue, so I'll explain in case anyone else comes across the problem.

Firstly, to answer my question above, the generated source files go here:

`/target/generated-sources/io/confluent/examples/streams/avro`

If you do encounter the problem, the solution is to find this folder in your IDE (I'm using IntelliJ in this case): `/target/generated-sources`, and mark it as a "generated sources root".  That sorted out my project in IntelliJ.

Now as to why you couldn't replicate it, @guozhangwang: I believe it's to do with the order you do things in.  If you do it in this order, the problem will **not** occur:
1. Clone the repo
1. Run `mvn clean install -Dmaven.test.skip=true`
1. Open the project in IntelliJ
1. Build the project in IntelliJ

However if you swap do steps 3 and 4 before 2, the problem will occur (at least it did in my case: IntelliJ on Ubuntu).  For some reason IntelliJ doesn't recognise that `generated-sources` folder as one that contains generated sources.  So the solution is simply to mark it as such (right-click on the folder -> Mark Directory as -> Generated Sources Root).
